dev/com.ibm.ws.microprofile.metrics.cdi.2.3/src/io/astefanutti/metrics/cdi23/MetricProducer.java

```diff
@@ -111,7 +111,7 @@ private static Timer timer(InjectionPoint ip, MetricRegistry registry, MetricNam
 
     @Produces
     private static SimpleTimer simpleTimer(InjectionPoint ip, MetricRegistry registry, MetricName metricName, MetricsExtension extension) {
-        Metadata metadata = metricName.metadataOf(ip, Timer.class);
+        Metadata metadata = metricName.metadataOf(ip, SimpleTimer.class);
```

Comment:
Do we need to change for `ConcurrentGauge`?

Reply:
yes, fixed as well